packages/contentful--app-scripts/lib/upload/create-app-upload.js

```diff
@@ -4,8 +4,11 @@ const { validateBundle } = require('./validate-bundle');
 const { showCreationError } = require('../utils');
 const { createClient } = require('contentful-management');
 
-async function createAppBundleFromFile(orgId, token, zip) {
-  const client = await createClient({ accessToken: token });
+async function createAppBundleFromFile(orgId, token, zip, userAgentApplication) {
+  const client = await createClient({
+    accessToken: token,
+    application: userAgentApplication ? userAgentApplication : 'contentful.cli',
```

Comment:
This is not the CLI. Should rather be `contentful.app-scripts` or similar.

This is the CLI: https://github.com/contentful/contentful-cli

Reply:
🙈 
changed it